docs/meta/Contributing.md

```diff
@@ -11,7 +11,7 @@ even need to know Git, you can do it all from the web interface!
 
 On each page, there's a button that takes you to the GitHub page for editing:
 
-!["Edit this page" link present on each wiki page](/images/contributing/edit-this-page.png)
+!["Edit this page" link present on each wiki page](/static/images/contributing/edit-this-page.png)
```

Comment:
These must be from `/images`, the `/static` directory is _only_ for use by the build tools.

Reply:
Ah, fair enough 👍